mmv1/products/filestore/examples/ansible/instance.yaml

```diff
@@ -24,6 +24,7 @@ task: !ruby/object:Provider::Ansible::Task
       - network: default
         modes:
           - MODE_IPV4
+        connectMode: DIRECT_PEERING
```

Comment:
I need to fix this too `connect_mode` too

Reply:
right @rileykarson ?

Reply:
Ansible isn't really supported by MM anymore (I need to clean up the rest of it, but it no longer gets generated/updated) so NBD either way